cli/internal/repository/ocm/pathmatcher.go

```diff
@@ -0,0 +1,89 @@
+package ocm
+
+import (
+	"context"
+	"fmt"
+
+	genericv1 "ocm.software/open-component-model/bindings/go/configuration/generic/v1/spec"
+	resolverspec "ocm.software/open-component-model/bindings/go/configuration/resolvers/v1alpha1/spec"
+	"ocm.software/open-component-model/bindings/go/credentials"
+	"ocm.software/open-component-model/bindings/go/plugin/manager"
+	pathmatcher "ocm.software/open-component-model/bindings/go/repository/component/pathmatcher/v1alpha1"
+	"ocm.software/open-component-model/bindings/go/runtime"
+	"ocm.software/open-component-model/cli/internal/reference/compref"
+)
+
+func convertToRaw(repository runtime.Typed) (*runtime.Raw, error) {
+	scheme := runtime.NewScheme()
+	scheme.MustRegister(repository, "v1")
+	raw := runtime.Raw{}
+	err := scheme.Convert(repository, &raw)
```

Comment:
fails with 

` failed to create get type for prototype of type runtime.Type: prototype not found in registry`
<img width="1222" height="284" alt="Screenshot 2025-09-26 at 14 26 31" src="https://github.com/user-attachments/assets/e770e60b-d4d0-49fc-a689-292a8a282e78" />